src/components/Counter/Counter.tsx

```diff
@@ -65,19 +66,25 @@ export class Counter extends React.Component<CounterProps> {
       onChange,
       value,
       error,
+      counterAriaLabel,
       ...rest
     } = this.props;
 
     return (
-      <div {...style('root', { disabled, error }, rest)} dir="ltr">
+      <div
+        {...style('root', { disabled, error }, rest)}
+        dir="ltr"
+        role="region"
+        aria-labelledby={counterAriaLabel}
```

Comment:
Just a note:
I think that if we use this the prop for `aria-labelledby` then we should probably change its name to `counterAriaLabelledBy`.
And while we're at it, we should probably also accept a `counterAriaLabel` prop to pass as a `aria-label` to the root div.
This way, developers using this component can use either one of these props